<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupUsersWidget</class>
 <widget class="QWidget" name="SetupUsersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="11">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6" rowspan="11">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="textLabel1_4">
     <property name="text">
      <string>Real Name:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="QLineEdit" name="realname"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="textLabel1_3">
     <property name="text">
      <string>User Name:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="4">
    <widget class="QLineEdit" name="username"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="textLabel1_2">
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="4">
    <widget class="QLineEdit" name="pass1">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Password (Again):</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="4">
    <widget class="QLineEdit" name="pass2">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="userIDCheck">
     <property name="text">
      <string>Custom User ID</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="userID">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>65000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QCheckBox" name="admin">
     <property name="text">
      <string>Add user to administrator (wheel) group</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="2">
    <widget class="QCheckBox" name="noPass">
     <property name="statusTip">
      <string>Not recommended</string>
     </property>
     <property name="text">
      <string>Bypass authorization attempts for this user</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="addMoreUsers">
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="5">
    <widget class="QWidget" name="advancedList" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <layout class="QGridLayout">
        <item row="1" column="1">
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout">
          <item>
           <widget class="QPushButton" name="editButton">
            <property name="text">
             <string>Edit Selected User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Delete Selected User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="autoLoginLabel">
            <property name="text">
             <string>Auto Login:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="autoLogin">
            <property name="minimumContentsLength">
             <number>22</number>
            </property>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QListWidget" name="userList"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Users:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="1" colspan="5">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="pass_error">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid white;
border-radius:6px;
background-color:white;
color:#5A2847;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="caps_error">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:1px solid white;
border-radius:6px;
background-color:white;
color:#5A2847;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="createButton">
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>realname</tabstop>
  <tabstop>username</tabstop>
  <tabstop>pass1</tabstop>
  <tabstop>pass2</tabstop>
  <tabstop>admin</tabstop>
  <tabstop>userIDCheck</tabstop>
  <tabstop>userID</tabstop>
  <tabstop>userList</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>autoLogin</tabstop>
 </tabstops>
 <resources>
  <include location="../../data.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>userIDCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>userID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addMoreUsers</sender>
   <signal>toggled(bool)</signal>
   <receiver>advancedList</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addMoreUsers</sender>
   <signal>toggled(bool)</signal>
   <receiver>userIDCheck</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addMoreUsers</sender>
   <signal>toggled(bool)</signal>
   <receiver>createButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addMoreUsers</sender>
   <signal>toggled(bool)</signal>
   <receiver>userID</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addMoreUsers</sender>
   <signal>toggled(bool)</signal>
   <receiver>cancelButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
